test/acceptance/features/bindAppToService.feature

```diff
@@ -585,3 +585,33 @@ Feature: Bind an application to a service
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-request-backend-a" should be changed to "False"
         And Secret "binding-request-backend-a" contains "BACKEND_READY" key with value "true"
         And Secret "binding-request-backend-a" contains "BACKEND_HOST" key with value "example.common"
+
+        @negative
+        Scenario: Create Service Binding with empty services
+        When Invalid Service Binding is applied
```

Comment:
We can reuse already existing step `Service Binding is applied`. What is the benefit of the new step?

Reply:
The new step has a different assertion expectations. The original step expect no error to be thrown, while the new step expect error to be thrown... so the behavior of each variant is different, therefore it should be split.